src/main.c

```diff
@@ -53,12 +85,15 @@ int print_usage_main()
     printf("Usage:\n\n");
     printf("    %sneural <command> [<args>]%s\n\n", BOLD, RESET);
     printf("Commands:\n\n");
-    printf("    %srun%s    Run inference using a trained network\n\n", BOLD, RESET);
+    printf("    %srun%s    Run inference using a trained network\n", BOLD, RESET);
+    printf("      %s[<path>]%s                    path to model (default: default.model)\n", BOLD, RESET);
```

Comment:
path is not a list. Shouldn't it be just <path>?

Reply:
`[]` means optional